priv/static/Linux.sh

```diff
@@ -0,0 +1,371 @@
+HOME="${HOME:-$(eval echo ~$USER)}"
+
+bold=$(tput bold)
+normal=$(tput sgr0)
+red='\033[0;31m'
+blue='\033[0;34m'
+bblue='\033[1;34m'
+white='\033[0;37m'
+green='\033[0;32m'
+cyan='\033[0;36m'
+
+function already_installed() {
+    case $1 in
+    "Zsh")
+        dpkg -l | grep -q zsh
+        ;;
+    "oh-my-zsh")
+        [ -d ~/.oh-my-zsh ]
+        ;;
+    "wget")
+        dpkg -l | grep -q wget
+        ;;
+    "Homebrew")
+        which brew >/dev/null 2>&1
+        ;;
+    "asdf")
+        brew list | grep -q asdf
+        ;;
+    "Erlang")
+        command -v erl >/dev/null 2>&1
+        ;;
+    "Elixir")
+        which elixir >/dev/null 2>&1
+        ;;
+    "Phoenix")
+        mix phx.new --version >/dev/null 2>&1
+        ;;
+    "PostgreSQL")
+        # which psql >/dev/null 2>&1
+        false
+        ;;
+    "Chrome")
+        dpkg -l | grep -q google-chrome-stable
+        ;;
+    "Node.js")
+        which node >/dev/null 2>&1
+        ;;
+    "ChromeDriver")
+        npm list -g | grep -q chromedriver
+        ;;
+    "Docker")
+        which docker >/dev/null 2>&1
+        ;;
+    *)
+        echo "Invalid name argument on checking"
+        ;;
+    esac
+}
+
+function install() {
+    case $1 in
+    "Zsh")
+        sudo apt-get install -y zsh
+        ;;
+    "oh-my-zsh")
+        sh -c "$(curl -fsSL https://raw.github.com/ohmyzsh/ohmyzsh/master/tools/install.sh)" "" --unattended
+        ;;
+    "wget")
+        sudo apt-get install -y wget
+        ;;
+    "Homebrew")
+        NONINTERACTIVE=1 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+        echo '# Set PATH, MANPATH, etc., for Homebrew.' >>~/.zshrc
+        (
+            echo
+            echo 'eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"'
+        ) >>~/.zshrc
+        eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
+
+        # recommendation after installing homebrew
+        brew install gcc
+        ;;
+    "asdf")
+        brew install asdf
+        (
+            echo
+            echo '. $(brew --prefix asdf)/libexec/asdf.sh'
+        ) >>~/.zshrc
+        source ~/.zshrc >/dev/null 2>&1
+        ;;
+    "Erlang")
+        sudo apt-get update
+        sudo apt-get -y install build-essential autoconf m4 libncurses5-dev libwxgtk3.0-gtk3-dev libwxgtk-webview3.0-gtk3-dev libgl1-mesa-dev libglu1-mesa-dev libpng-dev libssh-dev unixodbc-dev xsltproc fop libxml2-utils libncurses-dev openjdk-11-jdk
+        asdf plugin add erlang https://github.com/asdf-vm/asdf-erlang.git
+        asdf install erlang 25.1.2
+        asdf global erlang 25.1.2
+        asdf reshim erlang 25.1.2
+        ;;
+    "Elixir")
+        asdf plugin add elixir https://github.com/asdf-vm/asdf-elixir.git
+        asdf install elixir 1.14.2-otp-25
+        asdf global elixir 1.14.2-otp-25
+        asdf reshim elixir 1.14.2-otp-25
+        ;;
+    "Phoenix")
+        source ~/.zshrc >/dev/null 2>&1
+        mix local.hex --force
```

Comment:
Add `local.rebar --force` step and remove it from the Makefile.

Reply:
I think we need local.rebar --force on Makefile, the test fail because of that missing rebar3 on Makefile (Latest Github Action Result)

```
==> phx_tools_test
Could not find "rebar3", which is needed to build dependency :telemetry
I can install a local copy which is just used by Mix
** (Mix) Could not find "rebar3" to compile dependency :telemetry, please ensure "rebar3" is available
make: *** [test/scripts/Makefile:5: serve] Error 1
Error: Process completed with exit code 2.
```